Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM rails:4.2.6
+
+WORKDIR /upaya
+
+COPY Gemfile /upaya
+COPY Gemfile.lock /upaya
+RUN bundler install --without deployment deploy
+
```

Comment:
I think this is missing `bin/setup`, or at least the copying of the various yml files and `saml.key.enc`. When I ran `docker-compose run --rm web rake db:setup RAILS_ENV=development` on a fresh clone of the repo on my Mac, the rake task aborted with this error: `rake aborted!
Errno::ENOENT: No such file or directory @ rb_sysopen - /upaya/config/saml.key.enc`. 

Running `bin/setup` locally did not solve the issue.